app/Category.php

```diff
@@ -5,7 +5,7 @@
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Notifications\Notifiable;
 
-class Profile extends Model
+class Category extends Model
```

Comment:
We should create new models instead of renaming ones we no longer need, it causes conflicts this way.